async-nats/tests/jetstream_tests.rs

```diff
@@ -94,6 +102,51 @@ mod jetstream {
         );
     }
 
+    #[tokio::test]
+    async fn get_or_create_stream() {
+        let server = nats_server::run_server("tests/configs/jetstream.conf");
+        let client = async_nats::connect(server.client_url()).await.unwrap();
+        let context = async_nats::jetstream::new(client);
+
+        context.create_stream("events").await.unwrap();
+        assert_eq!(
+            context
+                .get_or_create_stream("events")
+                .await
+                .unwrap()
+                .info
+                .config
+                .name,
+            "events".to_string()
+        );
+
+        assert_eq!(
+            context
+                .get_or_create_stream(&StreamConfig {
+                    name: "events2".to_string(),
+                    ..Default::default()
+                })
+                .await
+                .unwrap()
+                .info
+                .config
+                .name,
+            "events2".to_string()
+        );
+    }
+
+    #[tokio::test]
+    async fn get_stream_bad() {
+        let server = nats_server::run_server("tests/configs/jetstream.conf");
+        let client = async_nats::connect(server.client_url()).await.unwrap();
+        let context = async_nats::jetstream::new(client);
+
+        assert_eq!(
+            context.get_stream("events").await.unwrap().info.config.name,
```

Comment:
This will panic.

```suggestion
            context.get_stream("events").await.unwrap().info.config.name,
```

Reply:
yeah, was a leftover to check error messages... fixed.